### PR TITLE
Precise version check

### PR DESCRIPTION
**Pull request by varshab1210**
**What type of PR is this?**

/kind enhancement

**What does this PR do / why we need it**:
Precise kam version check in the test feature file

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes N/A, directly saw it when I was checking the e2e test file

**How to test changes / Special notes to the reviewer**:

```make e2e``` test should pass in CI

### CODE REVIEW

**Comment by amitkrout** [comment]
/ok-to-test
@varshab1210 thanks for the e2e test contribution

File: test/e2e/features/basic.feature [hunk 1]
```diff
@@ -4,7 +4,7 @@ Feature: Basic test
     Scenario: KAM version
         When executing "kam version" succeeds
         Then stderr should be empty
-        And stdout should contain "kam version"
+        And stdout should contain "kam\sversion\sv\d+\.\d+\.\d+"
```
Comment (amitkrout) [inline test/e2e/features/basic.feature:7]: ```Then stdout should match "kam\sversion\sv\d+\.\d+\.\d+"```

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: test/e2e/features/basic.feature (hunk 1 above)
Comment (varshab1210) [reply test/e2e/features/basic.feature]: Done

**Comment by amitkrout** [comment]
Test log - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-developer_kam/186/pull-ci-redhat-developer-kam-master-v4.6-integration-e2e/1367740755442929664/artifacts/integration-e2e/integration-e2e-steps/build-log.txt
/lgtm
/approve

**Pull request merged**